core-server/src/main/java/org/glassfish/jersey/server/ChunkedOutput.java

```diff
@@ -290,6 +290,7 @@ public Void call() throws IOException {
         } finally {
             if (closed) {
                 try {
+                    closeByClient();
                     responseContext.close();
```

Comment:
@jansupol Is this proper way to propagate close from client to JerseyEventSink?

Reply:
how about to name it `ònClose()`?

Reply:
fixed